backend/src/main/java/com/bakdata/conquery/models/auth/web/DefaultAuthFilter.java

```diff
@@ -36,20 +40,33 @@
  */
 @Slf4j
 @PreMatching
-@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
 @Priority(Priorities.AUTHENTICATION)
-public class DefaultAuthFilter extends AuthFilter<AuthenticationToken, Subject> {
+@Provider
+public class DefaultAuthFilter extends AuthFilter<AuthenticationToken, Subject> implements Feature {
+
+	@Inject
+	@Setter
+	private IterableProvider<TokenExtractor> tokenExtractors;
 
-	private final Set<TokenExtractor> tokenExtractors = new HashSet<>();
+
+	public static DefaultAuthFilter asDropwizardFeature() {
+		final DefaultAuthFilter authFilter =
+				new Builder()
+						.setAuthenticator(new ConqueryAuthenticator())
+						.setUnauthorizedHandler(new DefaultUnauthorizedHandler())
+						.buildAuthFilter();
+		return authFilter;
+	}
```

Comment:
Müsen wir und auch noch mal anschauen. Das mit dem DropwizardFeature war immer irgendwie ein Umweg.
Ich weiß auch gerade nicht nicht warum es DynamicFeature in Jersey gibt, aber daher kam das glaube ich

Reply:
Habe gerade nachgeschaut, das mit dem DynamicFeature nutzen wir nicht mehr und DropwizardFeature gibt es auch nicht mehr. Die Methode kann als eigentlich umbenannt werden.